include/mbedtls/rsa.h

```diff
@@ -38,6 +38,60 @@
 #include "mbedtls/threading.h"
 #endif
 
+#if defined(PSA_CRYPTO_DRIVER_TEST)
+#include <psa/crypto_driver_common.h>
+#include <psa/crypto.h>
+
+typedef struct {
+    /* If non-null, on success, copy this to the output. */
+    void *forced_output;
+    size_t forced_output_length;
+    /* If not PSA_SUCCESS, return this error code instead of processing the
```

Comment:
This description is a bit misleading. You'll still process `something` - you'll set the forced output. And you'll return the status even if it's PSA_SUCCESS, if you had any forced output.

Reply:
1. If forced output is defined, then we return forced output + forced status (without processing).
2. If forced output is not defined (null) and forced status is not success, then return forced status (without processing).
3. Otherwise we do processing.

This comments are taken from other similar files e.g:
https://github.com/ARMmbed/mbedtls/blob/development/tests/include/test/drivers/aead.h#L29-L31